conanfile.py

```diff
@@ -69,5 +69,6 @@ def package(self):
 
     def package_info(self):
         self.cpp_info.libs = tools.collect_libs(self)
-        if self.settings.os == "Linux":
+        if self.settings.os == "Linux" or \
```

Comment:
Can't these lines be simplified to:
```suggestion
if self.settings.compiler in ["gcc", "clang"]:
```

Reply:
Yes, looks better, but is no longer required as termcap is a base requirement now.